sql/core/src/test/resources/sql-tests/results/ansi/datetime.sql.out

```diff
@@ -291,49 +291,47 @@ select timestamp '2019-01-01中文'
 -- !query
 select timestamp'2011-11-11 11:11:11' + interval '2' day
 -- !query schema
-struct<TIMESTAMP '2011-11-11 11:11:11' + INTERVAL '2 days':timestamp>
+struct<TIMESTAMP '2011-11-11 11:11:11' + 172800000000:timestamp>
```

Comment:
`Literal.toString` or/and `Literal.sql` should support new intervals.

Reply:
shall we fix it first? otherwise the changes here are hard to review.

Reply:
+1

Reply:
Here is the PR for this: https://github.com/apache/spark/pull/32196

Reply:
Merged the PR, rebased this PR and re-generated expected test results.